docs/DeveloperGuide.md

```diff
@@ -257,28 +257,46 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* Marcus is a 26 years old fresh graduate and has started working as a financial advisor.
+* He is well versed in tech and prefers a command-line interface for performing tasks.
+* Due to the nature of his work, he stores numerous client contacts on his google contacts, but dislikes it’s interface.
+* He dislikes mixing his professional and personal contacts.
+* He wants a way to store his clientele details in a separate place.
+* He likes to get things done fast.
+* He prefers typing over using a mouse.
+* He is reasonably comfortable using CLI apps.
```

Comment:
Shouldn't you remove this line considering you already have "* He is well versed in tech and prefers a command-line interface for performing tasks." ?